triton/tut/gpu.rst

```diff
@@ -312,19 +312,231 @@ on efficient data loading.
 
 .. _cuda-nvprof:
 
-Profiling GPU usage with nvprof
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Profiling GPU usage with NVIDIA Profilers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+NVIDIA provides a suite of profiling tools designed to help users optimize and analyze the performance of applications running on NVIDIA GPUs. These tools are essential for understanding how applications utilize GPU resources and for identifying bottlenecks in the code.
+
+Overview of NVIDIA Profilers:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Nsight Systems is a system-wide performance analysis tool that provides detailed insights into GPU workloads, CPU workloads, and their interactions. It helps identify issues related to CPU-GPU interaction, GPU utilization, memory bandwidth, and other system-level bottlenecks, such as I/O operations.
+
+Nsight Compute is a GPU kernel profiling tool specifically designed for deep-dive analysis of CUDA kernels. It provides a wide range of performance metrics.
+
+nvprof is being deprecated. It has been largely replaced by Nsight Compute and Nsight Systems, both of which provide a more comprehensive and user-friendly profiling experience.
```

Comment:
Anything about nvprof can be removed as the new documentation is better. It would make the text easier to read.